src/controllers/groupe.controller.ts

```diff
@@ -0,0 +1,83 @@
+import { NextFunction, Request, Response } from 'express';
+import { Group } from '@prisma/client';
+import { CreateGroupDto } from '@dtos/group.dto';
+import groupService from '@services/groups.service';
+
+class GroupController {
+  public groupService = new groupService();
+
+  public getGroups = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+      const findAllGroupsData: Group[] = await this.groupService.findAllGroup();
+
+      res.status(200).json({ data: findAllGroupsData, message: 'findAll' });
```

Comment:
I would create a generic Response type Dto with
ResponseDto<T> {
data?: T;
message?: string;
success: boolean;
error?: Error
}

and return normalized shape to the consumer like new ResponseDta({data: findAllGroupsData, success: true, message: 'findAll'}). 

Same for custom errors